### PR TITLE
add post totals

### PR DESCRIPTION
**Pull request by jhernandezb**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -149,8 +150,17 @@ fake-unstake:
 ###############################################################################
 proto-all: proto-gen proto-lint proto-check-breaking
 
+
 proto-gen:
-	@./contrib/protocgen.sh
+	@echo "Generating Protobuf files"
+	$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace tendermintdev/sdk-proto-gen sh ./contrib/protocgen.sh
+
+proto-format:
```
Comment (shanev) [inline Makefile:159]: 🔥

**Review by shanev — APPROVED** [review]
Looks good! This should save a lot of time in the end blocker when processing a lot of posts that end at the same time.

**Review by shanev — APPROVED** [review]
LGTM

**Review by JakeHartnell — APPROVED** [review]

**Comment by shanev** [comment]
Ready to merge?

**Comment by jhernandezb** [comment]
> Ready to merge?

ready now

**Pull request merged**